staging/src/kubevirt.io/api/flavor/register.go

```diff
@@ -22,4 +22,10 @@ package flavor
 // GroupName is the group name used in this package
 const (
 	GroupName = "flavor.kubevirt.io"
+
+	FlavorResourceSingularName = "virtualmachineflavor"
```

Comment:
Since you are using the package anyway, maybe you want to remove the `Flavor`?
```suggestion
	ResourceSingularName = "virtualmachineflavor"
```
Also I find `ResourceSingularName` a bit strange to read somehow, I'd prefer `SingularResourceName` or `ResourceNameSingular`, but that maybe just me.

Reply:
Done. Thanks!